src/Models/ProductModel.php

```diff
@@ -14,8 +13,58 @@ class ProductModel implements ProductModelInterface
      * ProductModel constructor.
      * @param $db
      */
-    public function __construct($db)
+    public function __construct(\PDO $db)
     {
         $this->db = $db;
     }
+
+    /**
+     * Gets all products from Database
+     * @return array|false depending on successful query or not
+     */
+    public function getAllProducts()
+    {
+        $query = $this->db->query('SELECT `sku`, `name`, `price`, `stockLevel` FROM `products`;');
+
+        if ($query){
+            return $query->fetchAll();
+        }
+
+        return $query;
+        }
+
+    /**
+     * @param ProductEntityInterface $productEntity
+     * @return bool if product has been added successfully to DB
+     */
+    public function addProduct(ProductEntityInterface $productEntity): bool
+    {
+        $array = [
+            "sku"=>$productEntity->getSku(),
+            "name"=>$productEntity->getName(),
+            "price"=>$productEntity->getPrice(),
+            "stockLevel"=>$productEntity->getStockLevel()
+        ];
+
+        $query = $this->db->prepare("UPDATE `products`
+                                    SET `sku` = :sku
+                                        `name` = :name
+                                        `price` = :price
+                                        `stockLevel` = :stockLevel
+        ");
+
+        return $query->execute($array);
+    }
+
+    /**
+     * @param string $sku
+     * @return bool whether the product was found in DB or not
+     */
+    public function checkProductExists(string $sku): bool
+    {
```

Comment:
This function was changed in story2 so maybe pull in the updated version before merging this